script/createrepo.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+# This script:
+
+# - creates a dockerfile
+# - prepares a docker image that can run `createrepo` that has the latest release rpms
+# - "runs" the image by creating a throwaay container
+# - copies the result of createrepo out of the throwaway container
+# - destroys the throwaway container
+mkdir -p createrepo/dist
+cat > createrepo/Dockerfile << EOF
+FROM fedora:32
+RUN yum install -y createrepo_c
+RUN mkdir /packages
+CMD touch /tmp/foo
+COPY dist/*.rpm /packages/
```

Comment:
This copies `rpm` files from the host filesystem into the container, bloating it and preventing container reuse.

Would it be better to instead mount the directory from the host into the container using Docker volumes? That way the `createrepo` command can operate on `rpm` files and create the necessary directory structure straight on the host, without having to copy anything to-and-from the container.

Something like:
```sh
## Dockerfile:
...
ENTRYPOINT ["createrepo"]

## usage on the host:
docker run --rm --volume "$PWD":/packages --workdir /packages createrepo "$@"
```